msu/classes/weighted_container.nut

```diff
@@ -35,6 +36,34 @@
 		this.Array.push(_item);
 	}
 
+	function find( _item )
+	{
+		foreach (i, item in this.Array)
+		{
+			if (item[1] == _item) return i;
+		}
+
+		return null;
+	}
+
+	function remove( _idx )
+	{
+		this.Array.remove(_idx);
```

Comment:
Should probably be `return this.Array.remove(_idx)` or `return this.Array.remove(_idx)[1]`. Which one is better?